docs/advanced/api_examples.yml

```diff
@@ -253,4 +254,9 @@
   response: |
     HTTP/1.1 200 OK
 
-    [{"dataset_id": "47440eff-e933-4de0-b56c-d3469b61421f", "primary": "%(NODE_0)s", "maximum_size": 1073741824, "metadata": {}, "deleted": false}]
+    [{"dataset_id": "47440eff-e933-4de0-b56c-d3469b61421f",
+      "primary": "%(NODE_0)s",
+      "maximum_size": 1073741824,
+      "metadata": {},
+      "deleted": false,
+      "path": "/flocker/47440eff"}]
```

Comment:
Maybe using a completely unrelated random byte sequence would be good, then, to avoid the appearance that what's going on is truncation.

This will make automatic example verification a little difficult later but I guess we'll figure out what to do about that when we actually get around to implementing that.